testlist/sim.dat

```diff
@@ -3,12 +3,14 @@
 
 # CONFIG_SIM_M32=y
 # The recent versions of macOS is 64-bit only
--sim:loadable
--sim:module32
--sim:sotest32
+-sim:loadable,Darwin
+-sim:module32,Darwin
+-sim:rpproxy,Darwin
+-sim:rpserver,Darwin
+-sim:sotest32,Darwin
 
 # X11
-# macOS doesn't have X11
+# macOS/Docker doesn't have X11
```

Comment:
Not for this PR, but I'll get the X11 headers added to the docker image.

Reply:
Dockerfile shows
```
  libx11-dev:i386 \
  libxext-dev:i386 \
```
I think these targets should build.

Reply:
So, I can enable it now?

Reply:
Done.